mmv1/third_party/terraform/website/docs/r/sql_user.html.markdown

```diff
@@ -119,7 +119,8 @@ The following arguments are supported:
 
 * `type` - (Optional) The user type. It determines the method to authenticate the
     user during login. The default is the database's built-in user type. Flags
-    include "BUILT_IN", "CLOUD_IAM_USER", "CLOUD_IAM_GROUP" or "CLOUD_IAM_SERVICE_ACCOUNT".
+    include "BUILT_IN", "CLOUD_IAM_USER", and "CLOUD_IAM_SERVICE_ACCOUNT" for both Postgres and MySQL.
```

Comment:
Would it make sense to link to the Postgres / MySQL docs for these fields?

Reply:
added